dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_28__Make_ou_event_and_enrollment_not_null.sql

```diff
@@ -0,0 +1,37 @@
+DO $$
+DECLARE dummyOrgUnitId bigint;
+DECLARE dummyOrgUnitUid varchar(11);
+BEGIN
+    select coalesce((select max(organisationunitid) + 1 from organisationunit), 1) into dummyOrgUnitId;
+    select generate_uid() into dummyOrgUnitUid;
+
+    while (select count(*) from organisationunit where uid = dummyOrgUnitUid) > 0 loop
```

Comment:
Maybe replace by count(1) or count(organisationunitid) or maybe even count(uid) since we are not interested in any of the actual columns. To make it pure index scan based.